great_expectations/dataset/dataset.py

```diff
@@ -4198,6 +4222,76 @@ def expect_column_kl_divergence_to_be_less_than(
 
         return return_obj
 
+    @MetaDataset.column_aggregate_expectation
+    def expect_column_pair_cramers_phi_value_to_be_less_than(
+        self,
+        column_A,
+        column_B,
+        bins_A=None,
+        bins_B=None,
+        n_bins_A=None,
+        n_bins_B=None,
+        threshold=0.05,
+        ignore_missings=None,
+        result_format=None,
+        include_config=True,
+        catch_exceptions=None,
+        meta=None,
+    ):
+        """
+        Expect the values in column_A to be independent of those in column_B.
+
+        Args:
+            column_A (str): The first column name
+            column_B (str): The second column name
+            threshold (float): Maximum allowed value of cramers V for expectation to pass.
+
+        Keyword Args:
+            bins_A (list of float): Bins for column_A if numeric.
+            bins_B (list of float): Bins for column_B if numeric.
+            n_bins_A (int): Number of bins for column_A if numeric. Ignored if bins_A is not None.
+            n_bins_B (int): Number of bins for column_B if numeric. Ignored if bins_B is not None.
+            ignore_missings (bool): \
+                If True, ignore columns where either column has missing. If False, treat missing values as
+                another category (only for methods ...).
+
+        Other Parameters:
+            result_format (str or None): \
+                Which output mode to use: `BOOLEAN_ONLY`, `BASIC`, `COMPLETE`, or `SUMMARY`.
+                For more detail, see :ref:`result_format <result_format>`.
+            include_config (boolean): \
+                If True, then include the expectation config as part of the result object. \
+                For more detail, see :ref:`include_config`.
+            catch_exceptions (boolean or None): \
+                If True, then catch exceptions and include them as part of the result object. \
+                For more detail, see :ref:`catch_exceptions`.
+            meta (dict or None): \
+                A JSON-serializable dictionary (nesting allowed) that will be included in the output without \
+                modification. For more detail, see :ref:`meta`.
+
+        Returns:
+            A JSON-serializable expectation result object.
+
+            Exact fields vary depending on the values passed to :ref:`result_format <result_format>` and
+            :ref:`include_config`, :ref:`catch_exceptions`, and :ref:`meta`.
+
+        """
+        crosstab = self.get_crosstab(
+            column_A, column_B, bins_A, bins_B, n_bins_A, n_bins_B
+        )
+        chi2_result = stats.chi2_contingency(crosstab)
+        # See e.g. https://en.wikipedia.org/wiki/Cram%C3%A9r%27s_V
+        cramers_V = np.sqrt(chi2_result[0] / self.get_row_count() / min(crosstab.shape))
```

Comment:
It looks to me like we are missing a constant adjustment here:

```suggestion
        cramers_V = np.sqrt(chi2_result[0] / self.get_row_count() / (min(crosstab.shape) - 1))
```